{"name": "test_Android", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable", "trace": "@pytest.mark.run(order=1)\n    @allure.severity(\"blocker\")\n    @allure.description(\"测试 小程序自动化疏通测试-Android\")\n    def test_Android():\n        startSmartXueWeiApp()\n>       mobileDriver(text='同意并继续').click()\nE       TypeError: 'str' object is not callable\n\ntest_flow\\test_Authentication\\test_jenkinsDebug.py:44: TypeError"}, "description": "测试 小程序自动化疏通测试-Android", "attachments": [{"name": "log", "source": "8fcfd597-2600-4427-aa98-2c3c9589a718-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "c0ddd2fd-e939-4bb5-8c0d-931c7cb2e60c-attachment.txt", "type": "text/plain"}], "start": 1637547908218, "stop": 1637547925979, "uuid": "d261e3ee-c6e0-4f0a-a466-05acced4a1e3", "historyId": "037f6ac01dca35d949e1d56271286eaa", "testCaseId": "faad5a259c6669bfb35d617fe89efcc0", "fullName": "test_flow.test_Authentication.test_jenkinsDebug#test_Android", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_flow.test_Authentication"}, {"name": "suite", "value": "test_jenkinsDebug"}, {"name": "host", "value": "BF-202011271053"}, {"name": "thread", "value": "1684-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_flow.test_Authentication.test_jenkinsDebug"}]}